main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"log"
+
+	"github.com/AlonSerrano/GolangBootcamp/handlers"
+
+	"github.com/labstack/echo/v4"
+)
+
+func main() {
+	e := echo.New()
+	v1 := e.Group("/api/v1")
```

Comment:
```suggestion
	group := e.Group("/api/v1")
```

I think we can find an even more descriptive name for this group.